### DISCUSSION TITLE
Streaming Hires and Lores stream

### DISCUSSION THREAD

**Discussion by mrlinux2u** [raspberrypi/picamera2#528] [Q&A]
Is there anyway (or any examples of) streaming both the hires and lores resolutions at the same time (ideally using ffmpegoutput) to a rtsp server?

I've googled and checked all the examples I can find and none of them show this method, I've even tried tweaking the available examples here and cannot for the life of me stream both resolutions at the same time.

Is this even possible as I'm having to use ffmpeg to resize my main resolution (1280x720) down to a smaller resolution (320x240 which gets fed to motioneye) and it's running the cpu for ffmpeg past 120% (for reference using raspivid / ffmpeg only uses approx 65% to do the same thing)?

Ideally if I can access both the hires & lores stream directly I can skip the ffmpeg resize part.

**Comment by davidplowman** [comment]
Hi, you should obviously be able to get both the main and lores streams out at whatever resolutions you want (subject to the lores being no larger than the main stream), so I would hope there would be no need for any extra resizing.

Currently Picamera2 only encodes one output stream, though that is something we could look at in future. If you wanted to encode a second stream then you'd have to do that one "by hand". I would expect you could output the h.264 bitstreams to pipes and get ffmpeg to remux/stream them from there?
